packages/frontend/src/components/RichTextEditor/ResizableImageExtension.tsx

```diff
@@ -0,0 +1,103 @@
+import { type CSSProperties, useCallback, useRef, useState } from 'react'
+import { RiSendToBack } from 'react-icons/ri'
+import TipTapImage from '@tiptap/extension-image'
+import {
+  type NodeViewProps,
+  NodeViewWrapper,
+  ReactNodeViewRenderer,
+} from '@tiptap/react'
+
+const MIN_WIDTH = 60
+const MAX_WIDTH = 750
+
+const ResizableImageTemplate = ({ node, updateAttributes }: NodeViewProps) => {
+  const imgRef = useRef<HTMLImageElement>(null)
+  const [resizingStyle, setResizingStyle] = useState<
+    Pick<CSSProperties, 'width'> | undefined
+  >()
+
+  const handleMouseDown = useCallback((event: React.MouseEvent) => {
```

Comment:
nit: can remove the `React.`

Reply:
This one cannot:
- it's to differentiate btwn React MouseEvent vs native MouseEvent
- also linting will behave weirdly if we import MouseEvent interface directly from 'react' 🤔